skyuxconfig.json

```diff
@@ -25,6 +25,7 @@
   "app": {
     "styles": [
       "@skyux/theme/css/sky.css",
+      "@skyux/theme/css/themes/modern/styles.css",
```

Comment:
Currently, Builder injects the `@skyux/theme/css/sky.css` style sheet automatically (even if the SPA lists it in skyuxconfig.json). Do you think we should update Builder to supply the modern stylesheet too?

Reply:
I want SPAs to be able to opt-in to supporting the new theme, so I'd prefer SPAs that don't support it to not even load the CSS. I guess we could do that via a config option in `skyuxconfig.json`.